src/main/java/com/nexmo/client/voice/Call.java

```diff
@@ -84,7 +103,7 @@ public void setFrom(Endpoint from) {
 
     @JsonProperty("answer_url")
     public String[] getAnswerUrl() {
-        return new String[]{answerUrl};
+        return (answerUrl == null) ? null : new String[]{answerUrl};
```

Comment:
This should probably be consistent with `getAnswerMethod`.

```suggestion
        return (answerUrl == null) ? new String[]{answerUrl} : null;
```